### PR TITLE
[dashboard] update schemadb metrics

### PR DESCRIPTION
**Pull request by lightmark**
## Motivation

Update all SchemaDB related metrics to new standard formats.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

Y

## Test Plan

n/a

### CODE REVIEW

File: terraform/templates/dashboards/storage.json [hunk 1]
```diff
@@ -1971,7 +2094,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "max(irate(libra_storage_ledger{type=\"events_created\"}[10m]) / on(peer_id) irate(libra_storage_latest_transaction_version[10m]))",
+          "expr": "max(irate(libra_storage_ledger{type=\"events_created\"}[1m]) / on(peer_id) irate(libra_storage_latest_transaction_version[1m]))",
```
[the remark this answers is not in the document]
Comment (msmouse) [reply terraform/templates/dashboards/storage.json]: (talked offline but realized the comment was till pending.)

[the remark this answers is not in the document]
Comment (lightmark) [reply terraform/templates/dashboards/storage.json]: I already updated it to 10m with `rate` in the latest commit.

[the remark this answers is not in the document]
Comment (msmouse) [reply terraform/templates/dashboards/storage.json]: yeah I saw

**Pull request closed without merging**